tmp/test-repo/README.md

```diff
@@ -0,0 +1 @@
+text test-repo30225782196400522
```

Comment:
I'm not sure if we can remove the `test-repo` folders. I need to check again how the tests run

Reply:
`tmp` should be ignored by `.gitignore` 💭